lib/kaffe/group_member/worker/worker.ex

```diff
@@ -0,0 +1,81 @@
+defmodule Kaffe.Worker do
+  @moduledoc """
+  A worker is assigned a single partition across topics. This is so we
+  effectively serialize the processing of any single key across topics.
```

Comment:
From the moduledocs I'm not clear on how a worker and a subscriber are different.